cloud_run_service_connect_cloud_sql/main.tf

```diff
@@ -0,0 +1,114 @@
+# [START cloudrun_service_cloudsql_dbuser_secret]
+resource "google_secret_manager_secret" "dbuser" {
+  project   = "your-project-id"
```

Comment:
We can remove the explicit project here and throughout. This will cause the test to fail since it will not detect the env var. Users can set this via env var https://cloud.google.com/docs/terraform/basic-commands or explicitly via provider initialization